packages/node_modules/@webex/internal-plugin-wdm/test/integration/spec/webex.js

```diff
@@ -5,33 +5,45 @@
 import '@webex/internal-plugin-wdm';
 
 import {assert} from '@webex/test-helper-chai';
-import sinon from 'sinon';
 import WebexCore from '@webex/webex-core';
 import testUsers from '@webex/test-helper-test-users';
 
-describe('plugin-wdm', function () {
-  this.timeout(30000);
+describe('plugin-wdm', () => {
+  describe('Webex', () => {
+    let device;
+    let user;
+    let webex;
 
-  let webex;
+    before('create users', () => testUsers.create({count: 1})
+      .then(([createdUser]) => {
+        user = createdUser;
+      }));
 
-  beforeEach('create users', () => testUsers.create({count: 1})
-    .then((users) => {
+    beforeEach('create webex instance', () => {
       webex = new WebexCore({
-        credentials: {
-          supertoken: users[0].token
-        }
+        credentials: user.token
       });
-      sinon.spy(webex.internal.device, 'unregister');
 
-      return webex.internal.services.waitForCatalog('postauth')
-        .then(() => webex.internal.device.register());
-    }));
+      device = webex.internal.device;
+    });
 
-  describe('onBeforeLogout()', () => {
-    it('unregisters the device', () => webex.logout({noRedirect: true})
-      .then(() => {
-        assert.called(webex.internal.device.unregister);
-        assert.isFalse(webex.internal.device.registered);
-      }));
+    describe('when the plugin is imported', () => {
+      it('should define a \'device\' plugin', () =>
```

Comment:
```suggestion
      it(`should define a 'device' plugin`, () =>
```

OR

```suggestion
      it('should define a "device" plugin', () =>
```

Reply:
A majority of our testing method descriptions in the JSSDK utilize `'` characters when mentioning strings as seen above. ESLint will throw if the ` character is used in a string without the usage of `${value}`. I can make a larger change to correct this, but this will break the style consistency of this test against all of the tests in the JS SDK.

Reply:
Not a big deal, just thought is unnecessary to escape `'` when there are other ways of achieving the same visual effect, that are also more pleasing to the eye in the code